app/assets/javascripts/kuroko2/job_instances.js

```diff
@@ -3,11 +3,28 @@
 
 jQuery(function ($) {
   var logIntervalId;
+  var notifyIfNeeded = function (status, name) {
+    if (!('Notification' in window)) {
+      return;
+    }
+
+    if (Notification.permission === 'granted' && Cookies.get('notification') === 'on') {
+      var notification = new Notification(
+        "[" + status + "] " + name,
+        {"icon": window.location.origin + "/assets/kuroko2/kuroko-logo-" + status.toLowerCase() + ".png"}
```

Comment:
It is not necessarily to mount kuroko2 engine on root.
How about using the rails generated path or URL in any data attribute?